deploy

```diff
@@ -66,6 +65,30 @@ get_secrets_for_usage
 rollout=()
 mkdir -p /tmp/kubernetes
 
+echo "Setting defaults"
+set_defaults
+
+if [ ! -d "$CI_PROJECT_DIR/kubernetes" ]; then
+  echo "No Kubernetes manifests found, using default manifest for buildpacks"
+  mkdir -p $CI_PROJECT_DIR/kubernetes
+  cp -r /opt/kubernetes-deploy/manifests/web/* $CI_PROJECT_DIR/kubernetes/
+  IFS=$'\n'
+  if cat Procfile | grep -v "^web:" | grep -v "^#" | grep ":" > /dev/null; then
+    ALL_STAGES=$(cat Procfile | grep -v "^web:" | grep -v "^#" | grep ":" | cut -d':' -f1)
+    for line in $ALL_STAGES; do
+      mkdir -p $CI_PROJECT_DIR/kubernetes/production
+      cp /opt/kubernetes-deploy/manifests/other/deployment.yaml $CI_PROJECT_DIR/kubernetes/deployment-$line.yaml
+      cp /opt/kubernetes-deploy/manifests/other/autoscale.yaml $CI_PROJECT_DIR/kubernetes/production/autoscale-$line.yaml
+      sed -i -e "s/{{PREFIX}}/$line/g" $CI_PROJECT_DIR/kubernetes/deployment-$line.yaml
+      sed -i -e "s/{{PREFIX}}/$line/g" $CI_PROJECT_DIR/kubernetes/production/autoscale-$line.yaml
+      set_prefix_defaults $line
+    done
+  fi
+fi
+
+echo "Setting defaults"
+set_defaults
```

Comment:
setting defaults two times?  I think I see why you need the 2nd one, but I don't understand the 1st one.

Reply:
I was thinking you would want to set_defaults AFTER set_prefix_defaults (which the IF statement does.)

Reply:
actually on second thought...one doesn't affect the other.  You're right...